assets/js/src/admin/popup-editor/general.js

```diff
@@ -56,6 +56,79 @@
 					}
 				}
 			});
+
+			document.querySelector( '#pum-popup-settings-container' ).addEventListener( 'click', function(e) {
+				if ( Array.from(e.target.classList).includes('popup-type') || Array.from(e.target.parentElement.classList).includes('popup-type') ) {
+					const $container = jQuery( '#pum-popup-settings-container' );
+					if ( 1 === $container.length ) {
+						// Temporarily hardcoded object for testing...
+						const popupTypes = {
+							'center-popup': {
+								'size': 'medium',
+								'responsive_min_width': '0%',
+								'responsive_max_width': '100%',
+								'animation_type': 'fade',
+								'animation_speed': 350,
+								'location': 'center',
+								'overlay_disabled': false,
+								'stackable': false,
+								'disable_reposition': false,
+							},
+							'left-bottom-notice': {
+								'size': 'tiny',
+								'responsive_min_width': '0%',
+								'responsive_max_width': '100%',
+								'animation_type': 'slide',
```

Comment:
I think this one should fade in, similar to how messages do in asana, or those "... so & so just purchased ...".

Reply:
Oh, I was going more of the banner style here, like the hello bars, which mostly slide in. I guess if someone was using this more of a notification, it would make sense to fade. I guess it depends on which you think is the more common use-case. I remember seeing people ask about hello-bar banner style popups more often but I would defer to whichever you think is best since you hear more from customers than I do.

Reply:
You say **hello bar** & **banner** and I think of the ones across the top of the screen, but you already have one of those in the presets.

Bottom left of screen is usually a **notification** of some kind, those tend to fade in & out.

Even hellobar.com has a full width top center banner bar as their primary offering: 
![image](https://user-images.githubusercontent.com/847818/77599240-7ef2f900-6eda-11ea-9f43-8806d8370a7b.png)

Reply:
@danieliser, yes, the hello bar or OptinMonster's floating bar or Quickie bar's top bar is exactly the use case my "Top Bar" preset was going after. For the bars/banners, I have found most of those tend to slide-in which is what I was saying in my previous comment.

Based on what I've seen, the use-cases where those fade in, like in Asana, as normally ones that are more notification/alert based. E.g, Something just saved.

Regardless, this thread was about the animation. Do you want the top bar to slide in or fade in?

Reply:
This comment was left in the code for bottom-left-notification ;p

The top bar can slide in or fade in, both seem fine to me.

But the bottom left notification should be a fade imho.

Reply:
@danieliser, Ah! That makes much more sense now. I was thinking this was the top bar and was confused about why you were getting confused 😂. Yes, this should definitely be fade. Not sure why I did slide there. Will update.